### PR TITLE
command_read: accept metric name with * globbing

### PR DESCRIPTION
**Pull request by rudy-6-4**
This change is [Reviewable](https://reviewable.io/reviews/criteo/biggraphite/144)

### CODE REVIEW

File: biggraphite/cli/command_read.py [hunk 1]
```diff
@@ -62,32 +65,45 @@ def run(self, accessor, opts):
...
+
+        metrics = [
+            accessor.get_metric(metric)
+            for metric in metric_names
+        ]
+
+        forced_stage = bg_accessor.Stage.from_string(opts.stage) if opts.stage else None
+
+        for i, metric in enumerate(metrics):
+            if i:
```
Comment (iksaif) [inline biggraphite/cli/command_read.py]: Move this to a sub function :)

File: biggraphite/cli/command_read.py [hunk 2]
```diff
@@ -62,19 +65,40 @@ def run(self, accessor, opts):
         See command.CommandBase.
         """
         accessor.connect()
-        metric = accessor.get_metric(opts.metric)
+        metric_names = (
+            accessor.glob_metric_names(opts.metrics) if '*' in opts.metrics else
+            [opts.metrics]
+        )
+        if not metric_names:
+            print("Globbing '%s' doesn't exist" % opts.metrics)
```
Comment (hlerebours) [inline biggraphite/cli/command_read.py]: Globbing pattern '%s' doesn't match any metric

File: biggraphite/cli/command_read.py [hunk 3]
```diff
@@ -62,19 +65,40 @@ def run(self, accessor, opts):
...
+            if i:
+                print()
+
+            self._display_metric(accessor, metric, metric_names[i], opts.time_start, opts.time_end, forced_stage)
+
+    @staticmethod
+    def _display_metric(accessor, metric, metric_name, time_start, time_end, forced_stage=None):
+        """
+        Print metric's information
+        """
```
Comment (hlerebours) [inline biggraphite/cli/command_read.py]: Must and with a period to pass the tests :)
"""Print metric's information."""

**Comment by iksaif** [comment]
Tests seem to be broken too

**Pull request merged**